ghost/core/core/server/adapters/storage/LocalFilesStorage.js

```diff
@@ -1,17 +1,51 @@
 // # Local File System Storage module
 // The (default) module for storing media, using the local file system
+const fs = require('fs-extra');
+const path = require('path');
 const config = require('../../../shared/config');
+
 const constants = require('@tryghost/constants');
+const urlUtils = require('../../../shared/url-utils');
 const LocalStorageBase = require('./LocalStorageBase');
 
+const messages = {
+    notFound: 'File not found',
+    notFoundWithRef: 'File not found: {file}',
+    cannotRead: 'Could not read File: {file}'
+};
+
 class LocalFilesStorage extends LocalStorageBase {
```

Comment:
Given that `LocalFilesStorage`, `LocalMediaStorage` and `LocalImagesStorage` are all __extremely__ similar now, we could probably move `saveRaw` up to `LocalStorageBase` and DRY up the implementation of `saveRaw`? 🙂

Reply:
Solid point. Done!